include/revng/Model/MiscellaneousConfiguration.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+/* TUPLE-TREE-YAML
+name: MiscellaneousConfiguration
+doc: |
+  Configuration options that don't belong anywhere else find themselves here.
+  As this list grows, they should be split into their own sub-sections.
+
+type: struct
+fields:
+  - name: EmitImpreciseCommentPositionWarning
```

Comment:
We need to review configuration options in there.

We need to add a configuration option only if we reasonably expect the user to actually customize it.
Configuration should never be a way to push onto the user a problem in our implementation.

Reply:
I agree. But it's always easier to remove a configuration option (i.e. hard code it as a `inline constexpr` variable in a header somewhere) that to add one.

My go-to is to add anything that I would want to customize and then, when we actually gather enough options for them to warrant some attention, just do a pass "demoting" some of them as a part of the inevitable hierarchy restructuring and whatnot.